ngsderive/commands/instrument.py

```diff
@@ -144,55 +138,48 @@ def resolve_instrument(
                 "unknown confidence",
                 "multiple instruments were detected in this sample",
             )
-        elif malformed_read_names_detected:
+        if malformed_read_names_detected:
```

Comment:
Were these changes by a linter/formatter? I find the code less easy to read here, but it's technically equivalent.

Reply:
Linter was complaining about the unnecessary `el`. I don't find one or the other any more difficult to read, so just sided with the linter on this.

I think this specific error came from the relatively new `Pylance`. It catches a lot of good things other linters don't, but it can be overzealous/opinionated at times. Overall I'm a fan of it though.